src/tabnineChatWidget/extensionCommands/commands.ts

```diff
@@ -0,0 +1,73 @@
+type SCOPE = "block" | "selection" | "none";
+export type Action = "test" | "fix" | "explain" | "document" | "workspace";
+export type Intent =
+  | "/generate-test-for-code"
+  | "/fix-code"
+  | "/explain-code"
+  | "/document-code"
+  | "/workspace";
+
+enum LensOrder {
+  test = 1,
+  fix = 2,
+  explain = 3,
+  document = 4,
+}
+
+export type Command = {
+  label: string;
+  text: Action;
+  intent: Intent;
+  description: string;
+  scope: SCOPE[];
+  multistep: boolean;
+  lensOrder?: LensOrder | undefined;
+};
+
+export const COMANDS: Command[] = [
```

Comment:
what about splitting it to `CODE_LENS_COMMANDS` and spread it into `COMMANDS`, so you won't need this lint ignore:
```
    COMANDS.filter(
      ({ text, lensOrder }) =>
        lensOrder && filterRelevantActions(text, location, diagnostics)
    )
      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
      .sort((a, b) => a.lensOrder! - b.lensOrder!)
```